package.json

```diff
@@ -17,11 +17,12 @@
     "@babel/plugin-transform-runtime": "^7.4.4",
     "@babel/plugin-syntax-dynamic-import": "^7.2.0",
     "@babel/preset-env": "^7.4.4",
-    "@babel/runtime": "^7.4.4"
+    "@babel/runtime": "^7.4.4",
+    "node-fetch": "2.5.0"
   },
   "devDependencies": {
     "typescript": "3.4.5",
-    "@types/node": "11.13.8",
+    "@types/node": "12.0.0",
```

Comment:
unfortunately aws lamda used by zeit now is still on 8.10 so that/s important to keep in mind